backend/src/graphql/model/User.ts

```diff
@@ -58,11 +58,11 @@ export class User {
 
   // `passphrase` text COLLATE utf8mb4_unicode_ci DEFAULT NULL,
 
+  @Field(() => Boolean)
+  isAdmin: boolean
```

Comment:
use date instead of boolean? We would have the info when the user was promoted to the admin